test/Microsoft.DotNet.Tools.Builder.Tests/IncrementalTests.cs

```diff
@@ -66,38 +64,59 @@ public void TestNoLockFile()
             Assert.Contains("does not have a lock file", buildResult.StdErr);
         }
 
-        [Fact(Skip="https://github.com/dotnet/cli/issues/980")]
         public void TestRebuildChangedLockFile()
         {
 
             var buildResult = BuildProject();
-            AssertProjectCompiled(_mainProject, buildResult);
+            AssertProjectCompiled(MainProject, buildResult);
 
-            var lockFile = Path.Combine(_tempProjectRoot.Path, "project.lock.json");
+            var lockFile = Path.Combine(TempProjectRoot.Path, "project.lock.json");
             TouchFile(lockFile);
 
             buildResult = BuildProject();
-            AssertProjectCompiled(_mainProject, buildResult);
+            AssertProjectCompiled(MainProject, buildResult);
         }
 
-        [Fact(Skip="https://github.com/dotnet/cli/issues/980")]
         public void TestRebuildChangedProjectFile()
         {
 
             var buildResult = BuildProject();
-            AssertProjectCompiled(_mainProject, buildResult);
+            AssertProjectCompiled(MainProject, buildResult);
 
-            TouchFile(GetProjectFile(_mainProject));
+            TouchFile(GetProjectFile(MainProject));
 
             buildResult = BuildProject();
-            AssertProjectCompiled(_mainProject, buildResult);
+            AssertProjectCompiled(MainProject, buildResult);
+        }
+
+        // regression for https://github.com/dotnet/cli/issues/965
+        [Fact]
+        public void TestInputHasSameTimeAsOutput()
```

Comment:
Can you change the name of the test to something like TestInputHasSameTimeAsOutputCausesProjectToCompile, or something like that? If you put a like I suggest, I believe you could even get rid of the big comment below.